### PR TITLE
Paragraph 3

### PR DESCRIPTION
**Pull request by pkosta8**
updated

### CODE REVIEW

**Comment by pkosta8** [comment]
done.

**Comment by jeffrothwell** [comment]
Great, love it

File: mad-libbing.txt [hunk 1]
```diff
@@ -7,10 +7,10 @@ She is the one in whom we can ______.
 
 She is the one named Sailorrrrrr ...
 
-Sailor ______!
-Sailor ______!
-Sailor ______!
-Sailor ______!
```
Comment (pkosta8) [inline mad-libbing.txt:13]: this is better

Comment (pkosta8) [inline mad-libbing.txt:13]: this is better

**Comment by vioif** [comment]
great lyrics!

**Pull request merged**